runners/flink/job-server/flink_job_server.gradle

```diff
@@ -165,23 +165,13 @@ task validatesPortableRunner() {
   dependsOn validatesPortableRunnerStreaming
 }
 
-project.ext.validatesCrossLanguageTransforms =
-  createPortableValidatesRunnerTask(
-    name: "validatesCrossLanguageTransforms",
-    jobServerDriver: "org.apache.beam.runners.flink.FlinkJobServerDriver",
-    jobServerConfig: "--clean-artifacts-per-job,--job-host=localhost,--job-port=0,--artifact-port=0,--expansion-port=0",
-    testClasspathConfiguration: configurations.validatesPortableRunner,
-    numParallelTests: 1,
-    pipelineOpts: [
-      // Limit resource consumption via parallelism
-      "--parallelism=2",
-      "--shutdownSourcesOnFinalWatermark",
-    ],
-    testCategories: {
-      // Only include cross-language transform tests. Avoid to retest everything on Docker environment.
-      includeCategories 'org.apache.beam.sdk.testing.UsesCrossLanguageTransforms'
-    },
-  )
-project.evaluationDependsOn ':sdks:python'
-validatesCrossLanguageTransforms.dependsOn ':sdks:python:setupVirtualenv'
-validatesCrossLanguageTransforms.systemProperty "pythonTestExpansionCommand", ". ${project(':sdks:python').envdir}/bin/activate && pip install -e ${project(':sdks:python').projectDir}[test] && python -m apache_beam.runners.portability.expansion_service_test"
+project.ext.validatesCrossLanguageRunner = createCrossLanguageValidatesRunnerTask(
+  jobServerDriver: "org.apache.beam.runners.flink.FlinkJobServerDriver",
```

Comment:
At least for Python we have to depend on `:runners:flink:1.5:job-server-container:docker` or bring up the FlinkJobServerDriver like we do in `flink_runner_test.py`.